app/src/main/java/com/codesoom/assignment/handlers/TaskHttpHandler.java

```diff
@@ -0,0 +1,287 @@
+        String title = body;
+        if(title.contains("=")){
+            title = getParameterFromBody(body);
+        }
+        responseCode = HttpStatusCode.BAD_REQUEST.getCode();
+        if (!body.isBlank() && split.length > 0 && split.length != 1) {
+            result = updateTask(title, extractIDFromBody(requestURIPath));
+        }
+
+        return result;
+    }
+
+    /**
+     * 사용자의 RequestMethod 가 PUT 였을 경우 Body Parameter 의 여부에 따라 필요한 메서드를 실행해주는 분기 메서드 입니다.
+     * @param requestURIPath 사용자가 접속 요청을 보낸 URI 에 path 정보가 String 인자로 들어옵니다.
+     * @return 분기를 통해 결과값을 String 으로 리턴합니다.
+     */
+
+    private String deleteMapper(String requestURIPath) throws IOException {
+        String result = ResultMessage.BAD_REQUEST.getMessage();
+        responseCode = HttpStatusCode.BAD_REQUEST.getCode();
+        String[] split = requestURIPath.split("/");
+        if (split.length > 0 && split.length != 1) {
+            result = deleteTask(extractIDFromBody(requestURIPath));
+        }
+        return result;
+    }
+
+    /**
+     * 저장되어있는 전체 Task 를 조회합니다.
+     * @return 전체 Task 를 String 으로 리턴합니다.
+     */
+
+    private String getTasks() throws IOException {
+        OutputStream outputStream = new ByteArrayOutputStream();
+        objectMapper.writeValue(outputStream, tasks);
+        return outputStream.toString();
+    }
+
+    /**
+     * ID 에 해당하는 Task 를 조회합니다.
+     * @param ID 조회할 Task 와 매칭될 ID 입니다.
+     * @return ID 에 해당하는 Task 를 JSON 형태의 String 으로 리턴하고, 해당되는 ID가 없으면 NOT_FOUND 메세지를 리턴합니다.
+     */
+
+    private String getTask(long ID) throws IOException {
+        OutputStream outputStream = new ByteArrayOutputStream();
+        if(tasks.size() > 0){
+            for(Task task : tasks){
+                if(task.getId() == ID){
+                    objectMapper.writeValue(outputStream, task);
+                    responseCode = HttpStatusCode.OK.getCode();
+                    return outputStream.toString();
+                }
+            }
+        }
+
+      responseCode = HttpStatusCode.NOT_FOUND.getCode();
+      return ResultMessage.NOT_FOUND.getMessage();
+    }
+
+    /**
+     * 새로운 Task 를 생성합니다.
+     * @param title Task 데이터가 들어있는 JSON 형태의 String 입니다.
+     * @return 전체 Task 를 리턴합니다.
+     */
+
+    private String setTask(String title) throws IOException {
+        Task taskJson = toTask(title);
+        if(tasks.size() > 0){
+            taskJson.setId(tasks.get(tasks.size() - 1).getId() + 1L);
+        }
+        tasks.add(taskJson);
+        responseCode = HttpStatusCode.CREATED.getCode();
+        return getTasks();
+    }
+
+    /**
+     * ID에 해당하는 Task 를 ArrayList 에서 찾아 title 을 업데이트합니다.
+     * @param title 매칭된 Task 의 변경 될 title 제목입니다.
+     * @param ID ArrayList 에 저장된 Task 를 매칭하기 위한 Key 값입니다.
+     * @return 매칭되는 ID가 있으면 해당 task 를 리턴하고, 없으면 Not Found message 를 리턴합니다.
+     */
+
+    private String updateTask(String title, long ID) throws IOException {
```

Comment:
method argument 에 대문자를 사용하지 않는 것이 관례입니다.
```suggestion
    private String updateTask(String title, long id) throws IOException {
```

Reply:
앗 감사합니다!